content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -57,8 +57,8 @@ created anew.
 
 {{< figure src="/images/docs/pod.svg" title="Pod diagram" class="diagram-medium" >}}
 
-*A multi-container Pod that contains a file puller and a
-web server that uses a persistent volume for shared storage between the containers.*
+A multi-container Pod that contains a file puller and a
+web server that uses a persistent volume for shared storage between the containers.
```

Comment:
This would work better as a caption for the figure. (Docs for the shortcode: https://gohugo.io/content-management/shortcodes/#figure).